src/components/Operatives/OperativeWorkOrderDetails.js

```diff
@@ -57,8 +76,35 @@ const OperativeWorkOrderDetails = ({ property, workOrder, personAlerts }) => {
           </GridColumn>
         </GridRow>
         <div className="lbh-heading-h5">Description</div>
-        <p className="govuk-body">{workOrder.description}</p>
-
+        <p
+          ref={pRef}
+          className={`govuk-body govuk-!-margin-bottom-0 ${
```

Comment:
Do we need the addition of `govuk-!-margin-bottom-0` class?

Reply:
if we don't want to have big gap between text and "show more/less", then yes